FIRO_TSEnsembles/src/main/java/hec/metrics/MetricCollection.java

```diff
@@ -62,7 +62,8 @@ public Duration getInterval() {
     //probably need getdataforparametername.
 
     public float[] getDateForStatistic(Statistics stat){
-        int index = Arrays.asList(metric_statisticsLabel).indexOf(stat);
+        String[] splitString = metric_statisticsLabel.split(",");
```

Comment:
Just checked the label for multiStatComputable and it looks like the new convention is to use a "|" to separate multiple stats in a multistat. So we should replace the "," here with the pipe.

Reply:
For the single string case, the pipe operator doesn't seem to work.  Add both the pipe operator and comma as possible separator options